packages/sdk/src/types/Validations.ts

```diff
@@ -22,6 +22,9 @@ export interface ValidationsValidateVoucherParams {
 	gift?: {
```

Comment:
Did you rebase your branch on top of master? I don't get why I see the same changes here which already have been applied in previous commit: bf4e2ff